fluentui_tablayout/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,9 @@
 <resources>
     <!--- Announces the name of the tab and its position in the list of tabs-->
     <string name="tab_content_description">\u0020 tab %1$d of %2$d</string>
+
+    <!-- Describes the Numbering of Pill Butons Inside PillBar -->
+    <string name="pill_button_number">Button Number</string>
```

Comment:
neednot create two strings. Create one string and use modifiers for variables like the string on line 4

Reply:
Ok, done